web/src/html.js

```diff
@@ -122,20 +127,33 @@ function buildBasicOgMetadata() {
 function buildClaimOgMetadata(uri, claim, overrideOptions = {}) {
   // Initial setup for claim based og metadata
   const { claimName } = parseURI(uri);
-  const claimTitle = escapeHtmlProperty(claim.title ? claim.title : claimName);
+  const { meta, value, signing_channel } = claim;
+  const fee = value && value.fee && (Number(value.fee.amount) || 0);
+  const tags = value && value.tags;
+  const media = value && (value.video || value.audio || value.image);
+  const source = value && value.source;
+  const channel = signing_channel && signing_channel.name;
+  const thumbnail = value && value.thumbnail && value.thumbnail.url;
+  const mediaType = source && source.media_type;
+  const mediaDuration = media && media.duration;
+  const creationTimestamp = meta && (meta.creation_timestamp || 0);
```

Comment:
Should do `value.release_time --> creation time --> 0`. We've been using release time, which works for newer claims.